hooks/javascript/load-test-convert-ts-to-js.sh

```diff
@@ -0,0 +1,9 @@
+# This should be used by load-test only,
+
+currentBranch=$(git rev-parse --abbrev-ref HEAD)
+
+if git diff-tree --no-commit-id --name-only -r HEAD..origin/$currentBranch | grep -q 'load-test/'; then
+    echo 'hahahha!'
+    cd load-test
+    npm run bundle
```

Comment:
Do we need the pre-commit hook to be that specific? Only looking at load test. I wonder if there is an existing pre-commit hook to run if there is a `pacakge.json` in the repo? Our standard is to have a `build` script in the `pacakge.json` instead of `bundle`

Reply:
This might fail when running in CI since you would first have to ensure there is an `npm` install

Reply:
Also, since this is only used in 1 repo today, you could define the pre-commit hook local to that repo until we know how to standardize this?

Reply:
@tagoro9 Good point, I added `npm install`. And I updated the script from `bundle` to `build`. I tested in the vehicle-search repo and it worked.

> I wonder if there is an existing pre-commit hook to run if there is a pacakge.json in the repo?
Not sure if I understand your question, but for vehicle-search, there is a package.json in the root level. But within `load-test/` folder there's its own package.json file for converting typescript and other stuff. 

I've posted a [thread](https://team-turo.slack.com/archives/C06EBKC8VL7/p1713380965715019) in #proj-load-testing channel discussing about different approaches of this ts -> js automation. The other way is to directly commit to the `.git/hooks` but after some research it didn't seem like we can commit anything under `.git`, it's auto ignored by git. Happy to hear your thoughts!

Reply:
Pre-commit lets you create hooks local to a repo without committing that folder